CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+3.27.0
+	- Added dmidecode to well known paths for Red Hat (ENT-12988)
```

Comment:
```suggestion
	- Added $(paths.dmidecode) for Red Hat (ENT-12988)
```

Might be better so folks don't have to translate/wonder about "well known paths". That phrase doesn't come up with much official in our docs or via duckduckgo.